### DISCUSSION TITLE
Setting up solang

### DISCUSSION THREAD

**Discussion by Johnny9164** [hyperledger-labs/solang#709] [Q&A] [answered]
Hi all! I am following this tutorial [solang thingy](https://solana-labs.github.io/solana-solidity.js/l) and when running the first command, "yarn docker" i get this. Any ideas?
![Capture](https://user-images.githubusercontent.com/96505092/159782420-dda86412-b0ae-42b4-b3a6-31e6f7573677.PNG)

**Comment by seanyoung** [comment, marked as answer]
I think the problem here is that the  instructions fail to mention that you need to clone the repo. I've amended the readme: https://github.com/solana-labs/solana-solidity.js/commit/63731bf4adea5cc73217d5d2c45c59d9cdc5e00d

Also from discord I gather the issue is solved. Please let me know if it is not.
